doc/reference/tt_cli/start.rst

```diff
@@ -0,0 +1,36 @@
+.. _tt-start:
+
+Starting a Tarantool instance
+=============================
+
+..  code-block:: bash
+
+    tt start INSTANCE
+
+``tt start`` starts the specified Tarantool :ref:`instance <admin-instance_file>`.
+
+Details
+-------
+
+``INSTANCE`` is a name of an instance from the ``instances_available`` directory
+specified in the :ref:`tt configuration file <tt-config_file_app>`. This can be:
+
+*   a name of an :ref:`instance file <admin-instance_file>` without the ``.lua`` extension.
+*   a name of a directory containing the ``init.lua`` file.
+
+
+Examples
+--------
+
+*   Start the ``app.lua`` instance from the ``instances_available`` directory:
```

Comment:
What is the default value for the `instances_available` directory? The user might have the same question when reading this document. Consider indicating the default on the config page.

Reply:
Right now it's the current working directory but it's not the final decision, please see [this comment](https://github.com/tarantool/doc/pull/3022#discussion_r920927960). We decided to skip this detail so that readers don't rely on it.